### PR TITLE
Fixes for windows subsystem for linux

### PR DESCRIPTION
**Pull request by nipinium**
Currently Sublime always throws `FileNotFoundError`.

### CODE REVIEW

**Comment by vladfaust** [comment]
I don't have WSL to test, but it looks OK for me.

File: Crystal.sublime-build [hunk 1]
```diff
@@ -2,8 +2,7 @@
 	"cmd": ["crystal", "run", "--no-color", "$file"],
 	"file_regex": "^Error in (?<filename>.+\\.cr):(?<line_number>[0-9]*):?(?<column_number>[0-9]*): (?<message>.+)$",
 	"selector": "source.crystal",
-	"quiet": true,
```
Comment (vladfaust) [inline Crystal.sublime-build]: Please leave it as is for now. It should be another issue to discuss whether to supress the output.

Comment (nipinium) [reply Crystal.sublime-build]: sorry I forgot to put that on

**Comment by vladfaust** [comment]
Thank you, @nipinium!

**Pull request merged**